corehq/motech/models.py

```diff
@@ -164,6 +168,84 @@ def last_token(self, token: Optional[dict]):
     def notify_addresses(self):
         return [addr for addr in re.split('[, ]+', self.notify_addresses_str) if addr]
 
+    @property
+    def is_throttled_connection(self):
+        return self.throttle_window is not None
+
+    def request_is_throttled(self):
+        if not self.is_throttled_connection:
+            return False
+
+        if self._should_throttle_request():
```

Comment:
Nit: Instead of returning a True or False based on `if self._should_throttle_request()`, you could also return the result of the function directly with `return self._should_throttle_request()` since the function returns a bool.

Reply:
Right! 

(I had some other logic that was here which I pulled out, then didn't realize this could be simplified as you said)